src/app/reporting/Engine.cpp

```diff
@@ -172,7 +172,9 @@ DataModel::ActionReturnStatus RetrieveClusterData(DataModel::Provider * dataMode
     reportBuilder.Checkpoint(checkpoint);
 
     DataModel::ActionReturnStatus status(CHIP_NO_ERROR);
-    AttributeValueEncoder attributeValueEncoder(reportBuilder, subjectDescriptor, path, version, isFabricFiltered, encoderState);
+    AttributeValueEncoder attributeValueEncoder = {
```

Comment:
This change is not necessary for this PR. Please undo to make the delta smaller.